api-reference/echo-node-api/key-config-api.md

```diff
@@ -0,0 +1,110 @@
+# Key Config API
+
+Echo console lets you add multiple private keys to an encrypted file.
+
+To enter this mode add the flag `--configure-keys` to Echo node.
+
+On the first run, when keys file wasn't created yet, you would need to specify a password which will be used to encrypt future keys. This is done by calling `set_password` command and entering your secret passphrase. You can change your password using the same command.
```

Comment:
When the keys file is already generated password should be entered to unlock the keys file using the set_password command